extensions/amp-image-viewer/0.1/amp-image-viewer.js

```diff
@@ -367,7 +368,7 @@ export class AmpImageViewer extends AMP.BaseElement {
 
         // Update translation and scaling
         this.updatePanZoom_();
-      },
+      },image);
```

Comment:
I noticed that this mutateElement has to happen on the image, and not the `<amp-image-viewer>` element itself. Otherwise we get a infinite loop where this function, `measure`, triggers `onLayoutMeasure`, which calls `measure`.

Reply:
/cc @alabiaga if you remove the reference to `image`, it will result in an infinite loop where the `resetImageDimensions_()` function is called on `onLayoutMeasure`, but causes `onLayoutMeasure` to be called.